changelog/unreleased/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+#### Features
+- Test changes to check renaming
```

Comment:
What about the PR links? They should be here because we've lost all the info about where this entry came from at this point, right?

Reply:
I added links to assembled changelog, thanks